deployment/ds-hostnet-split/03-envoy.yaml

```diff
@@ -81,7 +67,6 @@ spec:
         - $(CONTOUR_SERVICE_HOST)
         - --xds-port
         - $(CONTOUR_SERVICE_PORT)
-        - --statsd-enabled
```

Comment:
@rata please raise an issue to remove the `--statsd-enabled` flag from `cmd/contour` if its not longer needed.

Reply:
Oh hey, you already did, #1086. Thank you!

Reply:
Thank you :)